code-of-conduct/gr.md

```diff
@@ -76,3 +76,17 @@
 
 **Σας ευχαριστούμε που βοηθάτε να γίνει το PyGreece μια φιλόξενη και φιλική κοινότητα για
 όλους.**
+
+## Σχετικοί Σύνδεσμοι
+
+Ο Κώδικας Δεοντολογίας του PyGreece έχει εμπνευστεί από τις παρακάτω πηγές:
+
+- [The Python Software Foundation Code of Conduct](https://policies.python.org/python.org/code-of-conduct)
+- [The EuroPython Society Code of Conduct](https://www.europython-society.org/coc/)
+
+## Άδεια Χρήσης
+
+Ο Κώδικας Δεοντολογίας του PyGreece μπορεί διανέμεται κάτω από τους όρους της άδειας
+χρήσης
+[Creative Commons Attribution-ShareAlike 4.0 International License](https://creativecommons.org/licenses/by-sa/4.0/).
```

Comment:
Would it make sense to link to the Greek translation here?

```suggestion
[Creative Commons Attribution-ShareAlike 4.0 International License](https://creativecommons.org/licenses/by-sa/4.0/deed.el).
```

Could possibly translate the name, but I think the `en` version is considered industry standard.

Reply:
The title in the greek translation seems a bit odd right?

Reply:
I'd prefer the english content tbh. Not sure how the content on the greek page is generated. Not a strong opinion though.

Reply:
Yeah I'd prefer the `en` version too, feel free to resolve this